R/layer.r

```diff
@@ -125,6 +125,12 @@ layer <- function(geom = NULL, stat = NULL,
 
   # Warn about extra params and aesthetics
   extra_param <- setdiff(names(params), all)
+  # Take care of size->linewidth renaming in layer params
+  if (geom$rename_size && "size" %in% extra_param && !"linewidth" %in% mapped_aesthetics(mapping)) {
+    aes_params <- c(aes_params, params["size"])
+    extra_param <- setdiff(extra_param, "size")
+    cli::cli_inform("Using {.arg size} as {.arg linewidth}")
```

Comment:
I think this message needs to be a bit more clear that you should now use `linewidth` instead of `size`.

Reply:
How is this?
```
> ggplot(airquality) + geom_line(aes(Day, Ozone, group = Month, size = Month))
Renaming size aesthetic to linewidth
ℹ linewidth now has its own aesthetic instead of conflating it with size
ℹ While you can continue to use size it is highly advices to adapt your code to the new aesthetic
```